visualization/tools/librealsense_viewer.cpp

```diff
@@ -98,16 +115,6 @@ class LibRealSenseViewer
     LibRealSenseViewer (pcl::LibRealSenseGrabber& grabber)
     : grabber_ (grabber)
     , viewer_ ("LibRealSense Viewer")
-    , grabber_second_ (grabber)
```

Comment:
Yeah, it is for multiple grabber test. Because I suppose that the viewer is just an example of using librealsense grabber, and one grabber is enough. Besides, I checked the other grabber's viewer, they use only one grabber, I think it is better to keep in line with them.